GitUp/Application/Document.m

```diff
@@ -723,14 +727,15 @@ - (void)_updateStatusBar {
     _pullButton.enabled = [_mapViewController validateUserInterfaceItem:(id)_pullButton];
     NSRect frame = _pullButton.frame;
     if (isBehind) {
-      _pullButton.image = [NSImage imageNamed:@"icon_action_fetch_new"];
+      _pullButton.image = _mainWindow.isInDarkAppearance ? [NSImage imageNamed:@"ActionFetchNewDark"] : [NSImage imageNamed:@"ActionFetchNew"];
```

Comment:
Why doesn't NSImage handle this automatically with asset catalog light/dark modes?

Reply:
See the note in the PR description.

For deploying to 10.8, the compiled asset catalog doesn't contain images at all: it dumps them all with the appropriate names in the Resources directory so AppKit in 10.8 can find them. In this case, the dark representations simply aren't emitted into the final app bundle.

If that were to bump up one, for deploying to 10.9, the images would then all be in the asset catalog, but AppKit in 10.9 doesn't fully understand multiple image representations in the compiled asset catalog and will non-deterministically choose one.

10.10 is the first version to more or less forward-support all the image features of the asset catalog.